spec/requests/api/v2/providers/courses_spec.rb

```diff
@@ -2,7 +2,8 @@
 
 describe 'Courses API v2', type: :request do
   let(:user) { create(:user) }
-  let(:organisation) { create(:organisation, users: [user]) }
+  let(:user_without_terms_accepted) { create(:user, accept_terms_date_utc: nil) }
+  let(:organisation) { create(:organisation, users: [user, user_without_terms_accepted]) }
```

Comment:
You could move `user_without_terms_accepted` and this definition of `organisation` (with a `user_without_terms_accepted` into the `context` below). That would describe the context better there and avoid polluting this global context.